gradle.properties

```
@@ -1,4 +1,4 @@
-releaseVersion=v2.1-rc16.1.1
+releaseVersion=v2.1-rc16.2.0
```

should be
```suggestion
releaseVersion=v2.1-rc16.1.2
```
since it is the second version of the sequencer, based on the same version of the tracer